vignettes/subgroup.Rmd

```diff
@@ -156,6 +156,15 @@ formula_reduced
 #> sigma ~ 0 + time
 ```
 
+# Regularization
+
+This analysis assumes there is already sufficient justification for the choice of the subgroup variable in the dataset. Nonetheless, a subgroup may appear significant due to random chance alone, and the inherent statistical multiplicity of subgroup selection increases the risk of this false positive outcome. If this is not a concern in your specific situation, then direct mitigation may not be necessary. Otherwise, one possible strategy is to express prior skepticism of a treatment-by-subgroup interaction: that is, to shrink the treatment-by-subgroup interaction fixed effects toward zero in the prior. A horseshoe prior is one possible shrinkage technique.^[Please be aware that horseshoe cannot be applied to individual scalar parameters. It must apply to an entire parameter class as a whole. Please read <https://paul-buerkner.github.io/brms/reference/horseshoe.html> to adjust the hyperparameters for your specific use case.]
```

Comment:
That is an unfortunate limitation about horseshoe priors in brms. I was unaware of it. 

On this topic about the need for skepticism ... It may be an area where there is a clear difference between subgroup consistency assessments done as part of regulatory submissions after phase-3 and inferential subgroup analysis that is being done to potentially inform future study design, e.g. analyzing phase-2 data to plan phase-3. 

In the latter setting, some skepticism about the magnitude of (even overall) treatment effects is warranted, to mitigate selection bias due to the fact that such analyses are typically only done after "successful" phase-2's, so we are implicitly conditioning on having observed a preliminary efficacy signal there. Similarly, if someone was tempted to use a subgroup treatment effect estimated from phase-2 to run a phase-3 trial in this subgroup, even more skepticism is warranted, because we are also implicitly conditioning on whatever signal led us to select this particular subgroup. 

Granted, horseshoe priors are a blunt instrument for generating this kind of skepticism.

Reply:
I was disappointed too about the limitation of horseshoe in `brms` prior specification (as well as the convergence difficulties). And I think the topic of skepticism goes pretty deep, possibly too deep to cover in a vignette like this which is intended as a tutorial of the functionality of the package. So I removed the regularization for now in aa42bb61156ef37310fe852ac2e5894b73252240. We can always add it back in a separate PR.